### ISSUE TITLE
[Failing Test] capg-conformance-stable-k8s-master (ci-cluster-api-provider-gcp-make-conformance-stable-k8s-ci-artifacts)

### ISSUE THREAD

**Issue by droslean** [kubernetes/kubernetes#85333]
**Which jobs are failing**:
`capg-conformance-stable-k8s-master (ci-cluster-api-provider-gcp-make-conformance-stable-k8s-ci-artifacts)`

**Since when has it been failing**:
`13th Nov 19:30 PST`

**Testgrid link**:
https://testgrid.k8s.io/sig-release-master-informing#capg-conformance-stable-k8s-master

**Reason for failure**:
```console
 Waiting for SSH key to propagate.
scp: /var/log/pods: No such file or directory
scp: /var/log/containers: No such file or directory
ERROR: (gcloud.compute.scp) [/usr/bin/scp] exited with return code [1].
ERROR: (gcloud.compute.instances.add-access-config) Could not fetch resource:
 - At most one access config currently supported.
ERROR: (gcloud.compute.instances.add-access-config) Could not fetch resource:
 - At most one access config currently supported.
ERROR: (gcloud.compute.instances.add-access-config) Could not fetch resource:
 - At most one access config currently supported.
ERROR: (gcloud.compute.instances.add-access-config) Could not fetch resource:
 - At most one access config currently supported.
ERROR: (gcloud.compute.instances.add-access-config) Could not fetch resource:
 - At most one access config currently supported.
cluster.cluster.x-k8s.io "test1" deleted
Error from server (NotFound): clusters.cluster.x-k8s.io "test1" not found
kind delete cluster --name=clusterapi || true
Deleting cluster "clusterapi" ...
ERROR: (gcloud.compute.forwarding-rules.delete) Could not fetch resource:
 - The resource 'projects/gce-gci-upg-lat-1-4-ctl-skew/global/forwardingRules/test1-apiserver' was not found
ERROR: (gcloud.compute.target-tcp-proxies.delete) Some requests did not succeed:
 - The resource 'projects/gce-gci-upg-lat-1-4-ctl-skew/global/targetTcpProxies/test1-apiserver' was not found
ERROR: (gcloud.compute.backend-services.delete) Some requests did not succeed:
 - The resource 'projects/gce-gci-upg-lat-1-4-ctl-skew/global/backendServices/test1-apiserver' was not found
ERROR: (gcloud.compute.health-checks.delete) Could not fetch resource:
 - The resource 'projects/gce-gci-upg-lat-1-4-ctl-skew/global/healthChecks/test1-apiserver' was not found
Listed 0 items.
Listed 0 items. 
```

**Anything else we need to know**:
/cc @alenkacz @hasheddan @alejandrox1 
/sig cluster-lifecycle
/priority critical-urgent
/milestone v1.17

**Comment by dims** [comment]
https://prow.k8s.io/?job=ci-cluster-api-provider-gcp-make-conformance-stable-k8s-ci-artifacts seems green, though not sure why test-grid is not showing the latest over at entries https://testgrid.k8s.io/sig-cluster-lifecycle-cluster-api-provider-gcp#capg-conformance-stable-k8s-master

**Comment by liggitt** [comment]
test grid is showing green again. fixed by #85351

/close
